src/Views/BasicNumbersInput.js

```diff
@@ -0,0 +1,50 @@
+import { Console } from '@woowacourse/mission-utils';
+import throwError from '../Utils/throwError.js';
+import Rules from '../Utils/Rules.js';
+import Errors from '../Constants/Errors.js';
+import { InputComment } from '../Constants/display.js';
+
+const BasicNumbersInput = {
+  get: async () => {
+    const userInput = await Console.readLineAsync(InputComment.BASIC_NUMBERS);
+    return userInput;
+  },
```

Comment:
```suggestion
  get: () => {
    return Console.readLineAsync(InputComment.BASIC_NUMBERS);
  },
```
지금 로직에서는 이렇게 축약해도 무방할 것 같네요!